docs/src/architecture.md

```diff
@@ -159,6 +172,88 @@ Kubernetes cluster.
     coordinate PostgreSQL active/passive topologies across different Kubernetes
     clusters through a higher-level operator or management tool.
 
+### Reserving Nodes for PostgreSQL Workloads
+
+Whether you're operating in a multi-availability zone environment or, more
+critically, within a single availability zone, we strongly recommend isolating
+PostgreSQL workloads by dedicating specific worker nodes exclusively to
+`postgres` in production. A Kubernetes worker node dedicated to running
+PostgreSQL workloads is referred to as a **Postgres node** or `postgres` node.
+This approach ensures optimal performance and resource allocation for your
+database operations.
+
+This revision should help ensure that your message is clear and professional.
+
+!!! Hint
+    As a general rule of thumb, deploy Postgres nodes in multiples of
```

Comment:
@jsilvela is it clearer now? I have defined it above and in the "before you start" section.

Reply:
Sorry, I had to leave yesterday without finishing the review.
So, my confusion with this on reading is, is it meant as a Postgres instance? I know it's not, but maybe I'd say something like.

> As a general rule of thumb, deploy Kubernetes nodes dedicated to Postgres in multiples of

Reply:
The section (in the documentation page :) ) is dense enough that IMO we can be a bit verbose to help.

Reply:
If you saw my previous answer please ignore.

Reply:
It's very clear this new way @gbartolini .
But I'd delete

> This revision should help ensure that your message is clear and professional.

😄

Reply:
what an idiot :) - this is what happens when I am in a rush to get to the office :)